src/components/modal/addtostudio/presentation.jsx

```diff
@@ -49,7 +50,7 @@ const AddToStudioModalPresentation = ({
             <div className="addToStudio-modal-header modal-header">
```

Comment:
I just noticed we are still wrapping the title in another div — is that necessary or can those be absorbed into the `ModalTitle` component?

Reply:
That's the header bar, which some modals have. I think that would make more sense as a separate, ModalHeader component.

Reply:
There's lots more to refactor, trying to keep focused on what I need to move forward with Join Flow :)